lib/api/apiUtils/object/objectLockHelpers.js

```diff
@@ -19,8 +20,9 @@ function calculateRetainUntilDate(retention) {
     const date = moment();
     // Calculate the number of days to retain the lock on the object
     const retainUntilDays = days || years * 365;
```

Comment:
unrelated to this PR, but i wonder if the semantics of 'years' parameter here is correct with respect to leap years...